docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,163 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[local-dns-setup-macos]]
+== Local DNS Set-up for macOS
+
+While in offline mode recent versions of macOS does not send out dns queries, and the process for using the local dns server from {project} is a bit more involved then
+other OS.
+
+=== Enable tap devices
+
+Check for the presence of `tapX` (`tap0`, `tap1`, etc..) in `/dev` directory. If no `tap` device is present, you need to install the `tuntap` package.
```

Comment:
Make sure we provide a way for the user to check for the devices (even something as simple as `ls /dev | grep tap`).

```adoc
Check for the presence of `tap` devices in *_/dev_*:

----
$ ls /dev | grep tap
----

If no `tap` devices are present, install the *tuntap* package:

----
$ brew install tuntap
----
```

As a side note, do we have a way of installing `tuntap` if the user does not have Homebrew installed?

Reply:
Yes, you can download it from http://tuntaposx.sourceforge.net/download.xhtml